src/components/boardComponent/boardComponent.js

```diff
@@ -1,34 +1,34 @@
 import React, { Component } from 'react';
 import Square from '../squareComponent/squareComponent';
+import GameHeader from '../gameHeaderComponent/gameHeaderComponent';
 import './board.css'
 
 class Board extends Component {
 
-    render() {
+       render() {
         return (
-            <div className="Board Board_Row">
-                <div className="text instructions-text">
-                    <div className="row">
-                        <div className="board-grid col-lg-4 col-md-4 col-sm-4 col-xs-4">
-                            <Square />
-                            <Square />
-                            <Square />
+            <div> 
+                <GameHeader />
+                <div className="Board Board_Row">
+                        <div className="text instructions-text">
+                            <div className="row">
```

Comment:
I'm confused why the `className` for this `div` is `row`. A `row` contains a `board-grid`?

Reply:
It shouldn't be row, I can see why that's confusing. I initially used rows to build the board. Just changed the naming so it makes more sense.